### PR TITLE
Fix. Cron. Added serve buttons to check cron.

### PR DESCRIPTION
**Pull request by svfcode**
Task https://doboard.com/1/task/4874

### CODE REVIEW

File: cleantalkantispam.php [hunk 1]
```diff
@@ -396,6 +396,20 @@ public function onAfterInitialise()
                 $save_params['connection_reports'] = array('success' => 0, 'negative' => 0, 'negative_report' => null);
             }
 
+            // Serve buttons
+            if (isset($_POST['ct_serve_run_cron_sfw_send_logs']) && $_POST['ct_serve_run_cron_sfw_send_logs'] === 'yes') {
```
Comment (Glomberg) [inline cleantalkantispam.php:400]: Please add CSRF checking here.

File: cleantalkantispam.php [hunk 2]
```diff
@@ -396,6 +396,20 @@ public function onAfterInitialise()
...
+                /** @var \Cleantalk\Common\Cron\Cron $cron_class */
+                $cron_class = Mloader::get('Cron');
+                $cron_class = new $cron_class;
+                $cron_class->serveCronActions('sfw_send_logs', time() + 120);
+            }
+            if (isset($_POST['ct_serve_run_cron_sfw_update']) && $_POST['ct_serve_run_cron_sfw_update'] === 'yes') {
```
Comment (Glomberg) [inline cleantalkantispam.php:406]: Please add CSRF checking here.

**Review by Glomberg — CHANGES_REQUESTED** [review]

File: cleantalkantispam.php (hunk 1 above)
Comment (svfcode) [reply cleantalkantispam.php:400]: Already checked here https://github.com/CleanTalk/joomla3.x-4.x-antispam/pull/32/files/617a7a0e2fc8ff303b9b7a487fb7f463a03c71b1#diff-da34647e4d084d8b926ed7ebeed639f9badd0c7632f411f2b50e3c77dc4f89b7R300.

File: cleantalkantispam.php (hunk 2 above)
Comment (svfcode) [reply cleantalkantispam.php:406]: Already checked here https://github.com/CleanTalk/joomla3.x-4.x-antispam/pull/32/files/617a7a0e2fc8ff303b9b7a487fb7f463a03c71b1#diff-da34647e4d084d8b926ed7ebeed639f9badd0c7632f411f2b50e3c77dc4f89b7R300.

**Review by Glomberg — APPROVED** [review]

**Review by alexandergull — APPROVED** [review]

**Pull request merged**